packages/kotti-ui/source/kotti-navbar/KtNavbar.vue

```diff
@@ -72,32 +73,98 @@ export default defineComponent({
 		NavbarNotification,
 		NavbarQuickLink,
 	},
-	mixins: [clickaway],
 	props: makeProps(KottiNavbar.propsSchema),
 	setup(props: KottiNavbar.PropsInternal, { emit }) {
-		const mobileMenuToggle = ref(false)
+		const mobileMenuRef = ref<HTMLElement | null>(null)
+		const navbarRef = ref<HTMLElement | null>(null)
+		const tippyTriggerRef = ref<HTMLElement | null>(null)
+		const tippyInstanceRef = ref<Instance | null>(null)
 
 		provide(
-			IS_NAVBAR_NARROW,
-			computed(() => props.isNarrow),
+			KT_NAVBAR_CONTEXT,
+			computed(() => ({ isNarrow: props.isNarrow, theme: props.theme })),
 		)
+
+		useTippy(
+			navbarRef,
+			computed(() => ({
+				arrow: false,
+				appendTo: () => document.body,
+				content: mobileMenuRef.value ?? undefined,
+				interactive: true,
+				maxWidth: 'none',
+				offset: [0, 0],
+				onCreate(instance) {
+					tippyInstanceRef.value = instance
+				},
+				placement: 'bottom-start',
+				theme: props.theme
+					? `kt-navbar-${props.theme} kt-navbar-shadow`
```

Comment:
Florian: if there is a theme, you don't add `kt-navbar` as a standalone class 

IOW misisng `kt-navbar`